tests/course_test.py

```diff
@@ -130,15 +127,11 @@ def test_get_course_details(self):
         test_attribute = course_sections.attributes[0]
         self.assertTrue(isinstance(test_attribute, Attribute))
         self.assertEqual(test_attribute.attribute, "DSGE")
-<<<<<<< HEAD
         self.assertEqual(
             test_attribute.attribute_description, "*DSAS General Ed. Requirements"
         )
-=======
         self.assertEqual(test_attribute.attribute_description, "*DSAS General Ed. Requirements")
```

Comment:
This merge conflict was resolved by including both versions rather than just one

Reply:
Not sure I entirely did what you asked with this one, so do let me know if I screwed it up on my end. I got rid of the bottom line because it was redundant, and it's likely I forgot to delete it when cleaning up duplicates.

Reply:
Yup, that's precisely what I was referring to. The problem was that one version of the code had
```py
self.assertEqual(
    test_attribute.attribute_description, "*DSAS General Ed. Requirements"
)
```
while the other version of the code had
```py
self.assertEqual(test_attribute.attribute_description, "*DSAS General Ed. Requirements")
```
and git was showing both versions for you to pick one of the two, so one of them needed cleaning up.